### PR TITLE
Add the long form names in heading formatting.

### LINKED ISSUE #528: Add long form names in formatting dropdown menu
The __abreevs__ are cool in the label, but once expanded, we should show the full name. Take care localizing as I'm imaging the width will vary greatly. Also rename a few things, and more logically ordered the items too.

Note that `Aa` should appear when paragraphs are selected or when nothing is selected. This may be a change in behaviour, albeit just a label change.

notes-formatting-menu

### PR DESCRIPTION
**Pull request by Natim**
Fixes #528

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "@ckeditor/ckeditor5-dev-webpack-plugin": "^2.0.22",
     "@ckeditor/ckeditor5-editor-classic": "^1.0.0-alpha.2",
     "@ckeditor/ckeditor5-essentials": "^1.0.0-alpha.2",
-    "@ckeditor/ckeditor5-heading": "vladikoff/ckeditor5-heading#short-heading",
+    "@ckeditor/ckeditor5-heading": "Natim/ckeditor5-heading#short-heading",
```
Comment (Natim) [inline package.json]: Refs https://github.com/vladikoff/ckeditor5-heading/pull/1

File: locales/en-US/notes.properties [hunk 2]
```diff
@@ -41,6 +41,12 @@ syncComplete2=Synced at {time}
 # It is displayed below the 'synced' notes and above the 'local' notes. It is a visual separator.
 mergeWarning=**** On this computer: ****
 
+# Font size
+title1=Heading 1
```
Comment (Natim) [inline locales/en-US/notes.properties]: @ryanfeeley can you confirm that those are the label you want?

File: package.json (hunk 1 above)
Comment (vladikoff) [reply package.json]: Merged PR!

Comment (Natim) [reply package.json]: Should we move your repository to mozilla organization?

**Comment by Natim** [comment]
@flodolo I will be waiting for your approval and then merge.

**Comment by Natim** [comment]
Thanks

**Pull request merged**